security-release-process.md

```diff
@@ -196,7 +196,9 @@ security aspects to private channels. The fix lead will make the determination
 whether there would be user harm in handling the fix publicly that outweighs the
 benefits of open engagement with the community.
 
-Note: CVSS is convenient but imperfect. Ultimately, the fix lead has discretion
+Critical and High severity vulnerability fixes will receive an out-of-band release. Medium and Low severity vulnerability fixes will be released as part of the next Kubernetes [patch release](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md). 
```

Comment:
> Critical and High severity vulnerability fixes will receive

Maybe change to "will typically" or "will usually"?  Sometimes if the timing lines up, we might piggy-back on a release that's about to go out anyway.